recipe/meta.yaml

```diff
@@ -27,6 +27,7 @@ requirements:
   host:
     - boost-cpp
     - brotli
+    - bzip2
```

Comment:
Does it need to be added to the "run" dependencies as well? I forget whether we link statically by default...

Reply:
No, the `bzip2` package has a `run_exports` section that defines what goes into `run` of the package that uses it in the `host` section. To check this, see:

```
% conda render recipe | grep 'run:' -A 20 | grep bzip2
        - bzip2 >=1.0.8,<2.0a0
```

Reply:
Ah, is this a new conda thing?
But if we were to link statically, it would still appear as a runtime dependency?

Reply:
Yes, I guess it came with 4.6. 

If we link statically, it will still appear but we should link dynamically on conda-forge.

Reply:
Build output shows we're linking dynamically:

```
   INFO (arrow-cpp,lib/libarrow.so.14.1.0): Needed DSO lib/libbz2.so.1.0 found in conda-forge::bzip2-1.0.8-h516909a_1
```